test/updateDB.test.js

```diff
@@ -0,0 +1,197 @@
+const path = require('path')
+const fs = require('fs')
+const os = require('os')
+const fsify = require('fsify')({
+  cwd: os.tmpdir(),
+  persistent: false,
+  force: true
+})
+const execSync = require('child_process').execSync
+
+const updateDB = require('../updateDB')
+const BrowserslistError = require('../error')
+
+const root = process.cwd()
+const lastVersion = getLastVersion()
+
+beforeEach(() => {
+  jest.spyOn(console, 'log').mockImplementation(() => true)
```

Comment:
It will be better to mock `console.log` only in a specific test, since I may use `console.log` to debug some tests in the future.